docs/addons/aws-container-insights.md

```diff
@@ -0,0 +1,111 @@
+
+**IMPORTANT**
+
+CloudWatch does not automatically create all possible metrics from the log data, to help you manage your Container Insights costs. However, you can view additional metrics and additional levels of granularity by using CloudWatch Logs Insights to analyze the raw performance log events.
+
+Metrics collected by Container Insights are charged as custom metrics. For more information about [CloudWatch pricing](https://aws.amazon.com/cloudwatch/pricing/), see Amazon CloudWatch Pricing.
+
+Note: that this add-on can not co-exist with `adot-addon`, `cloudwatch-adot-addons` or `cloudwatch-logs` on same EKS cluster as they have conflicting and redundant interactions.
+
+## Usage
+
+Add the following as an add-on to your main.ts file to add CloudWatch Insights to your cluster
+
+```typescript
+import 'source-map-support/register';
+import * as cdk from 'aws-cdk-lib';
+import * as blueprints from '@aws-quickstart/eks-blueprints';
+
+const app = new cdk.App();
+
+const addOn = new blueprints.addons.CloudWatchInsights();
+
+const blueprint = blueprints.EksBlueprint.builder()
+  .version("auto")
+  .addOns(addOn)
+  .build(app, 'my-stack-name');
+```
+
+## Prerequisites
+
+Once the Container Insights add-on has been installed in your cluster, validate that the [AWS Cloudwatch Observability Controller]() is installed and FluentBit is installed: 
+
+```bash
+kubectl get pods -n amazon-cloudwatch
+```
+
+You should see output similar to the following respectively:
+
+```
+NAMESPACE           NAME                                                 READY   STATUS    RESTARTS        AGE
+amazon-cloudwatch   amazon-cloudwatch-observability-controller-manager   1/1     Running   1 (4d22h ago)   5d
+amazon-cloudwatch   cloudwatch-agent                                     1/1     Running   1 (4d22h ago)   5d
+amazon-cloudwatch   fluent-bit                                           1/1     Running   1 (4d22h ago)   5d
+```
+
+## View metrics for cluster and workloads
+
+Under Performance Monitoring, the Container Insights dashboard allows you to hone in on both cluster and workload metrics. After selecting EKS Pods and Clusters, you will see that the dashboard provides CPU and memory utilization along with other important metrics such as network performance.
+
+![CloudWatch](./../assets/images/eks-blueprint-cwinsights-performance-monitoring.png)
+
+## View cluster level logs
```

Comment:
Could we use doc and screenshots aligning to [the](https://aws.amazon.com/blogs/mt/new-container-insights-with-enhanced-observability-for-amazon-eks/) blog :
- Container Insights – Dashboard
- Top 10 list view
- Clusters overview
- Clusters performance monitoring
- CPU and memory utilization at the node level
- Containers performance monitoring
- Nodes performance monitoring
- Services performance monitoring
- Dashboard views for popular workloads
- New “ContainerInsights” metrics namespace
- Enhanced Container Insights – various metrics
- CW Logs
- EBS Performance
- Alarms?